.github/workflows/build.yml

```diff
@@ -0,0 +1,31 @@
+name: build
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.operating-system }}
+    strategy:
+      matrix:
+        operating-system: [ ubuntu-latest, windows-latest ]
```

Comment:
I've checked that all tests pass on macos as well, so feel free to add "macos-latest" here if you like.